Sources/Action/InputSubject.swift

```diff
@@ -23,7 +23,10 @@ public class InputSubject<Element>: ObservableType, Cancelable, SubjectType, Obs
 
     /// Indicates whether the subject has been isDisposed.
     public var isDisposed: Bool {
-        return _isDisposed
+        _lock.lock()
```

Comment:
Could probably just go for a 
`_lock.lock(); defer { _lock.unlock() }` 

If i'm not mistaken ?